Tribler/Test/Core/CreditMining/test_creditmining_sys.py

```diff
@@ -20,8 +20,9 @@
 from Tribler.Main.Utility.GuiDBTuples import CollectedTorrent
 from Tribler.Policies.BoostingManager import BoostingManager, BoostingSettings
 from Tribler.Test.Core.CreditMining.mock_creditmining import MockLtTorrent, ResourceFailClass
-from Tribler.Test.common import TORRENT_FILE
-from Tribler.Test.test_as_server import TestAsServer, TESTS_DATA_DIR
+from Tribler.Test.Core.Modules.Channel.base_test_channel import BaseTestChannel
```

Comment:
Actually, that's where I put common function that used in Both `Core.CreditMining` and `Core.Modules.Channel`. Should I put that, let say, in `Test.Core`?